google/cloud/pubsub/publisher_options.h

```diff
@@ -75,12 +75,32 @@ class PublisherOptions {
   }
 
   std::size_t maximum_message_count() const { return maximum_message_count_; }
+  /**
```

Comment:
Nit: The "maximum_hold_time" case inserts a blank line here.

Reply:
Fixed, thanks.